Determining if the pthread_create exist failed with the following output:
Change Dir: /home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_315be/fast"
/usr/bin/make -f CMakeFiles/cmTC_315be.dir/build.make CMakeFiles/cmTC_315be.dir/build
make[1]: ディレクトリ '/home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp' に入ります
Building C object CMakeFiles/cmTC_315be.dir/CheckSymbolExists.c.o
/usr/bin/cc     -o CMakeFiles/cmTC_315be.dir/CheckSymbolExists.c.o   -c /home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp/CheckSymbolExists.c
Linking C executable cmTC_315be
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_315be.dir/link.txt --verbose=1
/usr/bin/cc       CMakeFiles/cmTC_315be.dir/CheckSymbolExists.c.o  -o cmTC_315be -rdynamic 
CMakeFiles/cmTC_315be.dir/CheckSymbolExists.c.o: 関数 `main' 内:
CheckSymbolExists.c:(.text+0x38): `pthread_create' に対する定義されていない参照です
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_315be.dir/build.make:97: ターゲット 'cmTC_315be' のレシピで失敗しました
make[1]: *** [cmTC_315be] エラー 1
make[1]: ディレクトリ '/home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp' から出ます
Makefile:126: ターゲット 'cmTC_315be/fast' のレシピで失敗しました
make: *** [cmTC_315be/fast] エラー 2

File /home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp/CheckSymbolExists.c:
/* */
#include <pthread.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef pthread_create
  return ((int*)(&pthread_create))[argc];
#else
  (void)argc;
  return 0;
#endif
}

Determining if the function pthread_create exists in the pthreads failed with the following output:
Change Dir: /home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_11037/fast"
/usr/bin/make -f CMakeFiles/cmTC_11037.dir/build.make CMakeFiles/cmTC_11037.dir/build
make[1]: ディレクトリ '/home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp' に入ります
Building C object CMakeFiles/cmTC_11037.dir/CheckFunctionExists.c.o
/usr/bin/cc    -DCHECK_FUNCTION_EXISTS=pthread_create   -o CMakeFiles/cmTC_11037.dir/CheckFunctionExists.c.o   -c /usr/share/cmake-3.7/Modules/CheckFunctionExists.c
Linking C executable cmTC_11037
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_11037.dir/link.txt --verbose=1
/usr/bin/cc   -DCHECK_FUNCTION_EXISTS=pthread_create    CMakeFiles/cmTC_11037.dir/CheckFunctionExists.c.o  -o cmTC_11037 -rdynamic -lpthreads 
/usr/bin/ld: -lpthreads が見つかりません
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_11037.dir/build.make:97: ターゲット 'cmTC_11037' のレシピで失敗しました
make[1]: *** [cmTC_11037] エラー 1
make[1]: ディレクトリ '/home/pi/catkin_ws/build_isolated/genpy/CMakeFiles/CMakeTmp' から出ます
Makefile:126: ターゲット 'cmTC_11037/fast' のレシピで失敗しました
make: *** [cmTC_11037/fast] エラー 2


